test/e2etests.go

```diff
@@ -85,6 +86,48 @@ var tests = map[string]tester.ExoTest{
 				return err
 			}
 
+			return nil
+		},
+	},
+	"start-counter": {
+		FixtureDir: "start-counter",
+		Test: func(ctx context.Context, t tester.ExoTester) error {
+			// FIXME: This doesn't work right now because we don't have an easy way of
+			// removing volumes that are still attached to a previous container.
+			if _, stderr, err := t.RunCmd(ctx, "docker", []string{"volume", "rm", "e2etest-start-counter"}); err != nil {
+				if !strings.Contains(stderr, "No such volume") {
+					return err
+				}
+			}
+
+			if _, _, err := t.RunExo(ctx, "init"); err != nil {
+				return err
```

Comment:
Since this stuff is only used in tests, it's probably perfectly fine to simply panic on error inside `RunExo`.

Reply:
The intent was to allow people to create end to end tests where failure is expected — which becomes a bit ugly if they have to recover from panics all the time.